### ISSUE TITLE
Post-processing not included in run_inference.py 

### ISSUE THREAD

**Issue by plbenveniste** [ivadomed/utilities#23]
After running `nnUNetv2_find_best_configuration`, I get these instructions : 
<details>
<summary> inference_instructions.txt</summary>

```
***Run inference like this:***

nnUNetv2_predict -d Dataset444_zurich_mouse -i INPUT_FOLDER -o OUTPUT_FOLDER -f  0 1 2 3 4 -tr nnUNetTrainer -c 3d_fullres -p nnUNetPlans

***Once inference is completed, run postprocessing like this:***

nnUNetv2_apply_postprocessing -i OUTPUT_FOLDER -o OUTPUT_FOLDER_PP -pp_pkl_file ./nnUNet_results/Dataset444_zurich_mouse/nnUNetTrainer__nnUNetPlans__3d_fullres/crossval_results_folds_0_1_2_3_4/postprocessing.pkl -np 8 -plans_json ./nnUNet_results/Dataset444_zurich_mouse/nnUNetTrainer__nnUNetPlans__3d_fullres/crossval_results_folds_0_1_2_3_4/plans.json

```
</details>

The current `run_inference.py`doesn't perform post-processing. Should we add it ? Is there someone working on this ? (@naga-karthik @valosekj )

Also, I tried running the code on my computer: it took : 1277.31 seconds. 
Image is 80 MB (200x200x500) ; (0.05, 0.05, 0.05)

**Comment by naga-karthik** [comment]
No, I didn't included post-processing as it does not usually improve the performance that much. 

About this: 

> Also, I tried running the code on my computer: it took : 1277.31 seconds.

This is pretty long. Maybe you could only use the checkpoint from one fold only (assuming the results from the other folds does not vary much). Also, a rule of thumb to decide b/w `checkpoint_best` vs `checkpoint_final` can be found [here](https://github.com/MIC-DKFZ/nnUNet/issues/110#issuecomment-570148725)

**Comment by jcohenadad** [comment]
> Also, I tried running the code on my computer: it took : 1277.31 seconds.

😱 This is insanely long. Definitely not "packageable". I hope we can solve this

**Comment by plbenveniste** [comment]
I tried running it on the same image, on 1 fold with `best_checkpoint.pth` on cpu : 
Total time elapsed: 224.46 seconds
On 2 Images : (80 MB and 88.5 MB) both (200;200;500) and (0.05;0.05;0.05)
Total time elapsed: 400.05 seconds
